OpenRobertaServer/src/main/java/de/fhg/iais/roberta/javaServer/restServices/all/controller/ClientProgramController.java

```diff
@@ -344,7 +344,8 @@ public Response importProgram(@OraData HttpSessionState httpSessionState, JSONOb
                     UtilForREST.addSuccessInfo(response, Key.PROGRAM_IMPORT_SUCCESS);
                     Statistics.info("ProgramImport", "success", true);
                 } else {
-                    UtilForREST.addErrorInfo(response, Key.PROGRAM_IMPORT_ERROR_WRONG_ROBOT_TYPE);
+                    //Should find a way to get specific type of robot
+                    UtilForREST.addErrorInfo(response, Key.PROGRAM_IMPORT_ERROR_WRONG_ROBOT_TYPE).append("parameters", robotType1);
```

Comment:
Why is the robot type called parameters? You can select a better fitting name that represents the content of that key value pair.
You can also use `response.put` in a separate line to show clearly what is happening.

Reply:
Hi, please see my comments in the gci task.